src/dune_pkg/solver_env.mli

```diff
@@ -14,8 +14,7 @@ val get : t -> Variable_name.t -> Variable_value.t option
     existing values of those variables in [a]. *)
 val extend : t -> t -> t
 
-(** A [t] with a single variable "opam-version" set to the version of the opam
-    library dependen on by dune *)
-val with_opam_version_set_to_current : t
+(** A [t] with default values for some variables *)
+val with_defaults : t
```

Comment:
I actually like this name a lot better! Maybe mention the defaults in the documentation string?

Reply:
Added